### PR TITLE
fix(ListView): add class to set display: flex only for item with icon

### PR DESCRIPTION
**Pull request by akaseka**
**What is the problem this PR is trying to solve?**
listView styles issue
image

**What is the chosen solution to this problem?**
-Add class to set display: flex only for item with an icon

**Please check if the PR fulfills these requirements**

* [ ] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/src/ListView/Items/Item/Item.component.js [hunk 1]
```diff
@@ -41,6 +41,7 @@ class Item extends Component {
 			'checkbox',
 			{ 'switch-nested': children },
 			{ switch: isSwitchBox },
+			{'with-icon': item.icon},
```
Comment (inna-i) [inline packages/components/src/ListView/Items/Item/Item.component.js:44]:
```suggestion
			{ 'with-icon': item.icon },
```

**Review by inna-i — APPROVED** [review]

**Review by mkondzera — APPROVED** [review]

**Review by jmfrancois — CHANGES_REQUESTED** [review]
listview in form is broken
Screenshot 2020-11-16 at 13 42 10

**Comment by inna-i** [comment]
Looks good: 
image

**Review by inna-i — APPROVED** [review]

File: packages/forms/src/UIForm/fields/FieldTemplate/FieldTemplate.scss [hunk 2]
```diff
@@ -7,12 +7,6 @@
 		}
 
 		.checkbox {
-			&:after {
-				position: static;
-				display: inline;
-				content: '*';
-			}
-
```
Comment (jsomsanith-tlnd) [inline packages/forms/src/UIForm/fields/FieldTemplate/FieldTemplate.scss:15]: Don’t change a rule globally if the goal is to introduce a specific rule.
The idea is good: remove the star, but only in the ListView
- revert the css removal in FieldTemplate
- add a specific css in ListView style to override the global rule

**Review by lmaillet — APPROVED** [review]

**Review by jmfrancois — APPROVED** [review]

File: packages/forms/src/UIForm/fields/FieldTemplate/FieldTemplate.scss (hunk 2 above)
Comment (jsomsanith-tlnd) [reply packages/forms/src/UIForm/fields/FieldTemplate/FieldTemplate.scss:15]: Drop this suggestion after @frassinier's rework on style

**Pull request merged**